### DISCUSSION TITLE
Let chars on same account use portals

### DISCUSSION THREAD

**Discussion by arcosin** [chromiecraft/chromiecraft#7228] [Suggestions]
Normally, a character can only use a mage port if they are partied.
This is fine, but I think it would be nice for portals to be usable by other characters on your account.
Even if that means casting portal, re-logging to other character, and using. 
It would be a nice little quality-of-life thing that wouldn't take allot of difficulty to implement.

**Comment by andovcr** [comment]
This is doable if your two characters are in a party, correct?

If you have another player help group up your mage & alt, then you can create a portal on your mage, logout, login with your alt, and take the portal.  That worked in Classic Vanilla, so I'm assuming it still does - the only requirement being another player (or yourself on another account) creating the party with your mage and the character to be teleported.

**Comment by Nyeriah** [comment]
You can do as described by @andovcr 
The issue we had with not being able to use portals after players log out has been fixed.

Closing this one as its already possible without any further changes
